__mocks__/inquirer/prompt.mock.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
+'use strict';
+const Rx = require('rx');
+const inquirer = jest.genMockFromModule('inquirer');
+
+function prompt(questions, config, testAnswer) {
+ 	return questions.reduce(function (newOpts, ans) {
```

Comment:
Rx needs a return value and reduce needs to get the obj assign after we've gotten the answers